Dockerfile.qbicc

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:22.04
+LABEL maintainer="Ansu Varghese <avarghese@us.ibm.com>"
+
+#Dockerfile input is the address to start the JDWP server on
+ARG ADDRESS_ARG="0.0.0.0:8082"
+#Dockerfile input is the relative path of the debuggee directory
+ARG CLASS_NAME="Hello"
+ARG NATIVE_EXEC="apps/${CLASS_NAME}"
+ARG NATIVE_SRC="apps/${CLASS_NAME}sources"
+ARG IS_QUARKUS="false"
+ARG ASM_LINE="10"
+
+EXPOSE 8082
+HEALTHCHECK CMD wget -q -O /dev/null http://localhost:8080/healthy || exit 1
+
+WORKDIR /jdwp
+
+COPY startProcesses.sh .
+COPY target/NativeJDB-1.0-SNAPSHOT.jar .
+
+RUN export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -qqy update \
+    && apt-get -qqy install \
+    build-essential \
+    curl \
+    emacs \
+    valgrind \
+    vim \
+    libunwind-dev \
+    zlib1g-dev
+
+# TODO graalvm should not be needed
+COPY graalvm ./graalvm
+ENV GRAALVM_HOME /jdwp/graalvm
+ENV PATH $PATH:$GRAALVM_HOME/bin
+ENV JAVA_HOME $GRAALVM_HOME
+RUN $GRAALVM_HOME/bin/gu install native-image
```

Comment:
Let's remove these lines then?

Reply:
I left GRAALVM there bc java wouldn't be available when `startProcess.sh` is running

Reply:
oK I see...because none of the other JDKs worked.